docs/src/tutorial/tutorial.css

```
@@ -0,0 +1,54 @@
+ * Usage:  <svg class="inline-icon" viewBox="0 0 24 24" ...>...</svg>
+ *
+ * Sized to match the surrounding text, vertically centered on the
+ * text baseline, and colored with currentColor so they follow the
+ * mdBook theme automatically (Lucide uses stroke="currentColor").
+ */
+.inline-icon {
+  display: inline-block;
+  width: 1em;
+  height: 1em;
+  vertical-align: -0.125em;
+  stroke: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)